app/models/concerns/authorizable.rb

```diff
@@ -16,7 +16,7 @@ module Authorizable
       if user.nil?
         self.where('1=0')
       elsif user.admin?
-        self.scoped
+        self.where(nil)
```

Comment:
Been here before, haven't we?  I'm guessing it's why you have test failures:

https://github.com/theforeman/foreman/pull/2358#discussion-diff-32097442